compression/huffmancoding_test.go

```diff
@@ -0,0 +1,60 @@
+// huffmancoding_test.go
+// description: Tests the compression, encoding and decoding algorithms of huffmancoding.go.
+// author(s) [pedromsrocha](https://github.com/pedromsrocha)
+// see huffmancoding.go
+
+package compression
```

Comment:
Could you use the test package instead?
```suggestion
package compression_test
```